### ISSUE TITLE
Toolbar Documentation: Include example of toolbar existing in top subheader area when not attached to a component below

### ISSUE THREAD

**Issue by mceledonia** [patternfly/patternfly-next#1799]
This has come up a few times now and I've noticed its due to a gap in documentation:

![image](https://user-images.githubusercontent.com/30935938/57233414-94bead80-6fec-11e9-9c10-2c7ceedc130c.png)

When a toolbar is needed for a component that doesn't have an area for it to exist (e.g. grid view) the toolbar should be located at the bottom of the white background subheader area. The toolbar should never exist on it's own detached from anything as seen above.

We should include an example of the correct version of the above, and make sure to mention that it should never exist on it's own as a standalone item in the page.

**Comment by dongniwang** [comment]
Thanks @mceledonia for creating this issue!

Just wanted to add one more thing here - We also have a question about whether or not the toolbar should be sticky when scrolling the page. For a card view, we currently don't have pagination implemented, so if users start scrolling, the page title (sub-header) area will scroll and users will lose access of the toolbar, is there a recommendation for how to handle this? 

cc: @amysueg @seanforyou23

**Comment by amysueg** [comment]
Agree, thank you @mceledonia - and thanks in advance for your help addressing our questions!

**Comment by mcarrano** [comment]
@mceledonia Thanks for raising this issue. Agree that this should be added to the documentation here: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/toolbar  Would anyone be willing to take that on?

I am not so concerned about consistency here. I think users will see card grids vs. lists or tables as fundamentally different kinds of views. Interesting question about whether the sub-header (inc. the toolbar) should be sticky. Right now, looks like not of the header areas are sticky. @kybaker can you expand on the rationale for not doing this? I know it was discussed.

@dongniwang can you tell me more about how you want to use a toolbar on the context of the wizard? Are you saying that you need this card/grid view inside of a wizard?

**Comment by dongniwang** [comment]
@mcarrano - That's right. We have a full page wizard layout for one of our core user flows, and one of the wizard steps is to choose an item from a card view layout.

**Comment by mcarrano** [comment]
I think this ultimately becomes somewhat of a custom solution as the standard PF wizard comes in a modal. That said, there's no reason that couldn't be built using standard components. Do you have a mockup of this @dongniwang ? It would be good to explore how we might build this and/or if it requires anything new.
